frontend/src/metabase/search/components/InfoText/InfoText.styled.tsx

```diff
@@ -0,0 +1,27 @@
+import { css } from "@emotion/react";
+import styled from "@emotion/styled";
+import LastEditInfoLabel from "metabase/components/LastEditInfoLabel";
+import { breakpointMaxSmall } from "metabase/styled-components/theme";
+
+export const LastEditedInfo = styled(LastEditInfoLabel)`
+  ${({ theme }) => {
+    return css`
+      color: ${theme.colors.text[1]};
+      font-size: ${theme.fontSizes.sm};
+      font-weight: 500;
+
+      cursor: pointer;
+
+      &:hover {
+        color: ${theme.colors.brand[1]};
+      }
+    `;
+  }}
+  ${breakpointMaxSmall} {
```

Comment:
I need to stick to the more old-school CSS since we don't have the Mantine text components in here, so we can just throw `truncate` as a prop 🥲